applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig $(dvfs_state)_ENABLE
+	bool "Handle $(dvfs_state)"
+	default y
+	depends on $(dvfs_dependencies)
+
+if $(dvfs_state)_ENABLE
+
+config $(dvfs_state)_TIMEOUT_MS
+	int "timeout"
+	default 5000
+	range 0 65535
+	help
+	  Timeout in milliseconds specifying time after which DVFS request related to
+	  $(dvfs_state) times out. A request is automatically cleared on timeout.
+
+choice "$(dvfs_state)_ACTIVE_FREQ"
+	prompt "Default active frequency of $(dvfs_state)"
+	default $(dvfs_state)_FREQ_HIGH
+
+config $(dvfs_state)_FREQ_HIGH
+	bool "High frequency"
+
+config $(dvfs_state)_FREQ_MEDLOW
+	bool "Medium low frequency"
+
+endchoice
+
+config $(dvfs_state)_INITIAL_ACTIVE
```

Comment:
This Kconfig option seems unused and specific to the uninitializing state. Can we remove it?

Reply:
why unused? We use it for DVFS_STATE_INITIALIZING

Reply:
discussed offline.

Will consider remove config DESKTOP_DVFS_STATE_$(dvfs_state)_INITIAL_ACTIVE

We will then add code which sets DESKTOP_DVFS_STATE_INITIALIZING at the first module_state_ready event.
This will potentialy simplify kconfig template and make more clear approach in which we define dfvs_frequency = FREQ_MEDLOW etc. for each state